src/components/TopOptions/index.js

```diff
@@ -0,0 +1,62 @@
+import React, { PropTypes } from 'react';
+
+import {Row, Col} from 'react-bootstrap/lib';
+
+import InformationToggle from 'components/InformationToggle';
+import FontSizeDropdown from 'components/FontSizeDropdown';
+import TooltipDropdown from 'components/TooltipDropdown';
+import ReadingModeToggle from 'components/ReadingModeToggle';
+import Share from 'components/Share';
+
+const TopOptions = (props) => {
+  const {
```

Comment:
you can do that in the function argument. So instead of `(props) =>` you can do `({ options, surah, actions }) =>`